src/MDMTimingCurve.h

```diff
@@ -0,0 +1,152 @@
+/*
+ Copyright 2017-present The Material Motion Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <CoreGraphics/CoreGraphics.h>
+#import <Foundation/Foundation.h>
+#import <QuartzCore/QuartzCore.h>
+
+/**
+ The possible kinds of traits curves that can be used to describe an animation.
+ */
+typedef NS_ENUM(NSUInteger, MDMTimingCurveType) {
+  /**
+   The value will be instantly set with no animation.
+   */
+  MDMTimingCurveTypeInstant,
```

Comment:
Please assign an explicit numeric value to each of these.

Reply:
Done.